### ISSUE TITLE
GIO page refresh

### ISSUE THREAD

**Issue by mitchellryant** [TNRIS/tnris.org#1569]
Please add all content ideas and updates. @geochik 

- [ ] Event Calendar #1584 https://fullcalendar.io/download
- [ ] create banner @christoball

**Comment by mitchellryant** [comment]
![login-468](https://user-images.githubusercontent.com/8459928/40683690-d9ed8ae8-6354-11e8-84d7-a2c7a224c759.png)

**Comment by geochik** [comment]
@mitchellryant I added 3 time slots on my calendar to dedicate to the page design. I started.

**Comment by mitchellryant** [comment]
D+1

https://youtu.be/8xSFi-Tn2_g

**Comment by mitchellryant** [comment]
@geochik issue has been moved to backlog due to lack of everything.

**Comment by geochik** [comment]
@christoball   The attached jpg is a quick look at the rearrangement we would like to see on the GIO page. Let's discuss when you have a moment.
![Microsoft-Edge-Web-Notes-Texas-Geo506201859.jpg](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/599c64cafada7e0606ba7cba/125516c66e82c728ace21e0d46c59d99318d90a689fdf266a05af5d91e43253e070bb72e63c8d72da743684c280a11a9415e5d44fdf18c22b4aa2e30c80351b882294d7ed43818abba8719b655ee22de1a82b6bc8a4ec9ed3976e5de885d220c4749d7ab0268f9d31e8a4afc59e04a5fa5163562ef72357b469419702a792a.jpg)

**Comment by mitchellryant** [comment]
https://fullcalendar.io/download

**Comment by geochik** [comment]
@christoball  Please replace current photo for FDR with the attached photo on the GIO page.
![StatePhoto_FRetiz_2018.jpg](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/599c64cafada7e0606ba7cba/125516c66e82c728ace21e0d46db809b3787b3a180fdb07c826ff580205c18217955fb693cc7f33ab0487e021b4d44be44515846f9ff807ff2b469258d5c0bedd3645a70953f44abaa9e0dfd50e474d412dcfce9c85ecafa3c22ed8a8a4c3c1e041f81f65e31a2915985.jpg)

**Comment by christoball** [comment]
@geochik the photo file is missing.

**Comment by geochik** [comment]
![85_20180712_KW_0001.jpg](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/599c64cafada7e0606ba7cba/125516c66e82c728ace21e0d46b0c1a571d2d2f1dfbeee119b76c7ab791677226b48e13b628ae92dbc56395b4d5a47ab435d594da8ad8b7ef3b13c708a5e02e8d53e5d21c12f0dbefad341eb5ee779c11ac7a4b6984dc6e23c20e489894d3d1d021c8cfb5c27ed.jpg)

Let me know if this one works!

**Comment by christoball** [comment]
@geochik somethings going wrong with the upload.

**Comment by geochik** [comment]
Nope.....grab it here - M:\Admin\Staff\Felicia

**Comment by christoball** [comment]
@geochik Currently the mailchimp has 'email' 'first name' and 'last name' as the fields.

I think that they should be combined into just 'name' and add an 'organization' field instead.

**Comment by christoball** [comment]
A preview of page refresh so far:
https://preview.tnris.org/gio-page-refresh/d6877ece7efa2a503ae6472e672f9c02b287d243/geographic-information-office/

**Comment by christoball** [comment]
I need a short bio for @geochik . Please refer to the length of Richard's

**Comment by mitchellryant** [comment]
@geochik you could just write something and paste it into the issue.

**Comment by geochik** [comment]
@mitchellryant  Thank you!
Ms. Retiz works closely with the GIS community to establish strategic targets and goals for GIS initiatives in Texas. Felicia began her career in GIS at TNRIS in 1999 after receiving a Geography degree from the University of Texas at Austin.

**Comment by mitchellryant** [comment]
issue is waiting on comments\mockup of ideas from RW.

**Comment by adambreznicky** [comment]
https://preview.tnris.org/gio-page-refresh/a5093b5e995d25a8869f3901043fe7b0173d8707

preview was re-built. will be alive for a couple weeks, then it will die.

**Comment by geochik** [comment]
I created a Google calendar for GIS events in Texas. This is the first attempt at a draft. Let's see what this looks like on the page. I can play around with the dimensions but not much else. Thanks for your help, @mitchellryant!

<iframe src="https://calendar.google.com/calendar/embed?title=GIS%20Events%20Calendar&amp;mode=WEEK&amp;height=250&amp;wkst=1&amp;bgcolor=%23ffffff&amp;src=stategio.texas%40gmail.com&amp;color=%23333333&amp;ctz=America%2FChicago" style="border-width:0" width="800" height="250" frameborder="0" scrolling="no"></iframe>

**Comment by geochik** [comment]
@christoball Can I please have a rebuild of the latest preview page to work with?  Thank you!

**Comment by geochik** [comment]
@christoball Please add the following text to the latest update of the GIO page:

The Texas Geographic Information Officer (GIO) is designated by statute (Texas Water Code §16.021) to be the current Director of the Texas Natural Resources Information System (TNRIS). Establishing the GIO at TNRIS is appropriate given its history as the coordinating office for geospatial data collection, creation and maintenance since 1972. By having the responsibilities of the GIO defined in state statute, it clearly establishes authorized articulation of all matters associated with the operation of the office.
The most important function of the GIO is to build and maintain relationships with public agencies, institutes of higher education, private sector and trade associations. Additionally, the role of the GIO is to encourage cooperation between stakeholders to maximize the value of GIS data and services. The GIO achieves these charges by bringing the GIS community together to create connections, finding common needs, and building support for continued innovation to promote the use and adoption of geographic information systems.

I will request feedback when I receive the preview link. Thank you!

**Comment by mitchellryant** [comment]
@christoball @geochik 
Are we on check to publish by 01/31, as stated in the Jan 10 email.?
If not what is holding us back?

**Comment by geochik** [comment]
@mitchellryant 
Thank you for checking on this rapidly approaching deadline. I will require an additional week to work on some content and get  it reviewed before the final push. 
The reason for the delay was due to the "Event Calendar" requirement for the page. @christoball is looking for a way to make the calendar "dynamic" so that an entry to an online calendar, such as Google Calendar, can be embedded into the page. In the interest of time, we have made the decision to make it "manual" entry until the issue is resolved.

**Comment by adambreznicky** [comment]
@christoball as per your request from yesterday afternoon about moving the gio-news into a subdirectory of the geographic-information-office folder with the intent to alter the news url paths, i spent time yesterday and this morning looking into it. this seems to be a larger scale, more significant change in the gulpfile and cannot be completed quickly (from what i can tell). restructuring the fundamental folder structure and gulp build is out of scope for the time i have available to dedicate to this issue right now -- more significant projects are occuring. this may be something we can consider during the tnris.org rebuild in the future. all of this considered, i suggest leaving the current url path format as it currently stands.

**Comment by mitchellryant** [comment]
@geochik @christoball New publish date if 02/07

**Comment by christoball** [comment]
@geochik Can we change the three fields for the GIO Newsletter to the following:

Full Name
Organization
Email

**Comment by christoball** [comment]
@geochik preview of latest page:
https://preview.tnris.org/gio-page-refresh/6fbd187c1c5fecc85c129ae58658cb8349ae64de/geographic-information-office/

**Comment by christoball** [comment]
@geochik made adjustments to the top calendar with input from @rwade2126 

We felt that it was confusing for it not to all be in chronological order, so we highlighted the meeting in a different way:
https://preview.tnris.org/gio-page-refresh/94fdc99ecddde1b03e1a930f27fb6bf2af2949ae/geographic-information-office/

**Comment by christoball** [comment]
Got final approval to push from @geochik 

resolved in 179ea8d37fc7d009e1f61dd005924da994224032
